From 62da56ec55ea63aa1979d3bcf407bbf06eba1b1b Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Mon, 8 Jul 2024 00:05:24 +0200
Subject: [PATCH 1186/1695] drivers: clk: smt32mp15: stm32mp1_clk_on[] is local

Add static keyword to stm32mp1_clk_on[] array since it is local
to clk-stm32mp15.c clock driver.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git c64a0e1b4fffc06c9d498f21e6d030af4569dad5 drivers: clk: smt32mp15: stm32mp1_clk_on[] is local]
Change-Id: Ic5b17603e7385695866c008da1e0d27893565ba4
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/392004
Domain-Review: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 core/drivers/clk/clk-stm32mp15.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/drivers/clk/clk-stm32mp15.c b/core/drivers/clk/clk-stm32mp15.c
index cf4977c68..0c1727ac5 100644
--- a/core/drivers/clk/clk-stm32mp15.c
+++ b/core/drivers/clk/clk-stm32mp15.c
@@ -472,7 +472,7 @@ static const struct stm32mp1_clk_gate stm32mp1_clk_gate[] = {
 };
 DECLARE_KEEP_PAGER(stm32mp1_clk_gate);
 
-const uint8_t stm32mp1_clk_on[] = {
+static const uint8_t stm32mp1_clk_on[] = {
 	CK_HSE, CK_CSI, CK_LSI, CK_LSE, CK_HSI, CK_HSE_DIV2,
 	PLL1_P, PLL1_Q, PLL1_R, PLL2_P, PLL2_Q, PLL2_R, PLL3_P, PLL3_Q, PLL3_R,
 	CK_AXI, CK_MPU, CK_MCU,
-- 
2.47.2

